relationalize/schema.py

```diff
@@ -150,6 +151,40 @@ def drop_null_columns(self) -> int:
             del self.schema[column]
         return len(columns_to_drop)
 
+    def drop_special_char_columns(self) -> int:
+        """
+        Drops columns which have a non alnumeric (excluding whitespace) in their name from the schema.
+
+        Returns the # of columns that were dropped.
+        """
+        columns_to_drop = []
+        for key in self.schema.keys():
+            if re.sub(r"[^a-zA-Z0-9 ]", "", key) != key:
```

Comment:
this is a weird way to express what you want here I imagine find or match would be better

Reply:
changed to a more readable equation